WireUI/Package.swift

```diff
@@ -35,6 +38,18 @@ let package = Package(
         .target(name: "WireMainNavigationUI"),
         .testTarget(name: "WireMainNavigationUITests", dependencies: ["WireMainNavigationUI"]),
 
+        .target(name: "WireMoveToFolderUI", dependencies: ["WireFoundation"]),
+
+        .target(
+            name: "WireMoveToFolderUISupport",
+            dependencies: ["WireMoveToFolderUI"],
+            plugins: [
+                .plugin(name: "SourceryPlugin", package: "SourceryPlugin")
+            ]
+        ),
+
```

Comment:
optional: remove the empty lines, so that `WireMoveToFolderUI`, `WireMoveToFolderUISupport` and `WireMoveToFolderUITests` are grouped together.

Reply:
Done!